vision/objectdetection/detector.go

```diff
@@ -37,27 +37,53 @@ func Build(prep Preprocessor, det Detector, post Postprocessor) (Detector, error
 // Detection returns a bounding box around the object and a confidence score of the detection.
 type Detection interface {
 	BoundingBox() *image.Rectangle
+	NormalizedBoundingBox() []float64
 	Score() float64
 	Label() string
 }
 
 // NewDetection creates a simple 2D detection.
-func NewDetection(boundingBox image.Rectangle, score float64, label string) Detection {
-	return &detection2D{boundingBox, score, label}
+func NewDetection(imageBounds, boundingBox image.Rectangle, score float64, label string) Detection {
```

Comment:
I think this is a breaking change that should be announced as such, but I'm not confident of that. What do you think?

Reply:
I think it is not and is not lol but we've had this discussion enough now.